utilities/src/main/java/io/syndesis/qe/utils/DbUtils.java

```diff
@@ -78,42 +84,21 @@ public int executeSQLGetUpdateNumber(String sqlCommnad) {
      * <p>
      * getNumberOfRecordsInTable(myTable, desiredColumn, desiredValue)
      *
-     * @param tableName
+     * @param tableName table name
      * @param args check example
-     * @return
+     * @return number of records in the table
      */
     public int getNumberOfRecordsInTable(String tableName, String... args) {
         if (!(args.length == 0 || args.length == 2)) {
             throw new IllegalArgumentException("Incorrect usage of this method.");
         }
-
-        int records = 0;
-        try {
-            String sql = "SELECT COUNT(*) FROM " + tableName.toUpperCase();
-
-            if (args.length == 2) {
-                //be specific
-                sql = "SELECT COUNT(*) FROM " + tableName.toUpperCase() + " WHERE " + args[0] + " LIKE '" + args[1] + "'";
-            }
-            log.info("SQL: *{}*", sql);
-            final ResultSet resultSet = executeSQLGetResultSet(sql);
-
-            while (resultSet.next()) {
-                records = resultSet.getInt(1);
-            }
-        } catch (SQLException ex) {
-            log.error("Error: " + ex);
-        }
-        log.debug("Number of records: " + records);
-
-        return records;
+        String sql = "SELECT * FROM " + tableName.toUpperCase() + (args.length == 2 ? " WHERE " + args[0] + " LIKE '" + args[1] + "'" : "");
```

Comment:
While I do admit this is a bit clearer than the original, I'd say we should probably have two overloaded versions of this method.

Reply:
:+1: